genai-perf/genai_perf/measurements/run_config_measurement.py

```diff
@@ -70,7 +70,7 @@ class RunConfigMeasurement:
 
     def __init__(
         self,
-        gpu_metrics: GpuRecords,
+        gpu_metrics: GpuRecords = {},
```

Comment:
```suggestion
        gpu_metrics: GpuRecords,
```
I would not recommend setting the default values of the function parameters to be a mutable object. AFAIK, the default values in Python are evaluated only once and reused across the subsequent calls to the function, which could lead to an unexpected behavior:
```python
r1 = RunConfigMeasurement()
r1.gpu_metrics['x'] = 10
r2 = RunConfigMeasurement()
print(r2.gpu_metrics)  # output: {'x': 10}
```

Reply:
That's a very good point. I've fixed this so that the default is now None.